blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/ContainerFilter.java

```diff
@@ -58,7 +59,7 @@
                 }
                 filters[i] = f;
             }
-            Collection<T> out = new ArrayList<>();
+            Collection<T> out = new LinkedList<>();
```

Comment:
@vivek ordering of the collection does not survive when passed through this function